spec/models/role_spec.rb

```diff
@@ -9,7 +9,7 @@
     it "contains the correct values" do
       expected_roles = %w[
         admin banned chatroom_beta_tester codeland_admin comment_banned podcast_admin pro restricted_liquid_tag
-        single_resource_admin super_admin tag_moderator mod_relations_admin tech_admin trusted
+        single_resource_admin super_admin tag_moderator mod_relations_admin tech_admin support_admin trusted
```

Comment:
I took a look at the Travis failure and recreated it locally and what I found was that the spec is failing due to the order of this array--if you place`support_admin` before `tech_admin` to mirror the array in `role.rb`, this will pass!:
```
expected_roles = %w[
        admin banned chatroom_beta_tester codeland_admin comment_banned podcast_admin pro restricted_liquid_tag
        single_resource_admin super_admin tag_moderator mod_relations_admin support_admin tech_admin trusted
        warned workshop_pass
      ]
```